recipes/diligent-core/all/conanfile.py

```diff
@@ -58,6 +58,8 @@ def validate(self):
                     self.name, self._minimum_cpp_standard, self.settings.compiler, self.settings.compiler.version))
         if self.settings.compiler == "Visual Studio" and "MT" in self.settings.compiler.runtime:
             raise ConanInvalidConfiguration("Visual Studio build with MT runtime is not supported")
+        if self.version == "250014" and self.settings.compiler == 'Visual Studio' and self.settings.build_type == 'Debug':
+            raise ConanInvalidConfiguration("Can't build Visual Studio in Debug mode")
```

Comment:
Can we try removing this? I had the impression it was only a test but it should pass 🤞 given my result locally

```suggestion
```